test/dotnet-test.UnitTests/GivenATestCommand.cs

```diff
@@ -14,10 +14,13 @@ namespace Microsoft.Dotnet.Tools.Test.Tests
     public class GivenATestCommand
     {
         private static readonly string ProjectJsonPath = Path.Combine(
-                AppContext.BaseDirectory,
+                Directory.GetCurrentDirectory(),
```

Comment:
Is this right?  Do other tests do this?  I thought the idea was these assets get copied into the test directory, so the tests can build/publish/etc and not affect other tests.